tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TypeSpec/CustomizedCodeUpdateTool.cs

```diff
@@ -184,82 +185,139 @@ private async Task<CustomizedCodeUpdateResponse> UpdateCoreAsync(string commitSh
             return (false, ex.Message);
         }
     }
-    private async Task<(bool Success, bool RequiresReview)> ValidateAndUpdateGuidanceAsync(
-        string packagePath,
-        LanguageService languageService,
-        List<string> guidance,
-        CancellationToken ct)
+
+
+    /// <summary>
+    /// Builds the SDK project to validate compilation.
+    /// </summary>
+    private async Task<PackageOperationResponse> BuildSdkAsync(string packagePath, LanguageService languageService, CancellationToken ct)
+    {
+        try
+        {
+            // Validate inputs
+            if (string.IsNullOrEmpty(packagePath))
+            {
+                logger.LogError("Package path is null or empty in BuildSdkAsync");
+                return PackageOperationResponse.CreateFailure("Package path is required.");
```

Comment:
We don't need to create response result for the build method if the result isn't processed by the caller.